Drivers/DriverFactory.php

```diff
@@ -25,21 +25,29 @@ class DriverFactory
     protected $dbDriver;
 
     /**
+     *
+     * @var MongoDBDriver
+     */
+    protected $mongoDriver;
+
+    /**
      * @var TranslatorInterface
      */
     protected $translator;
 
     const DATABASE_DRIVER = 'Lexik\Bundle\MaintenanceBundle\Drivers\DatabaseDriver';
+    const MONGODB_DRIVER  = 'Lexik\Bundle\MaintenanceBundle\Drivers\MongoDBDriver';
 
     /**
      * Constructor driver factory
      *
      * @param DatabaseDriver      $dbDriver The databaseDriver Service
+     * @param MongoDBDriver       $mongoDriver The MongoDBDriver Service
      * @param TranslatorInterface $translator The translator service
      * @param array               $driverOptions Options driver
      * @throws \ErrorException
      */
-    public function __construct(DatabaseDriver $dbDriver, TranslatorInterface $translator, array $driverOptions)
+    public function __construct(DatabaseDriver $dbDriver, MongoDBDriver $mongoDriver, TranslatorInterface $translator, array $driverOptions)
```

Comment:
I don't think that injecting both drivers is a good idea, the best would be to refactor the `DriverFactory` to pass a "StorageDriverInterface" (or something like) that could be an ORM or MongoDB driver. And so in the `MongoDBDriver` you would not have to set the mongodb registry as nullbale, which a bit weird to me as this parameter is required to make the driver work.

And DoctrineMongodbBundle should be in the `suggest` section in the `composer.json`.